### ISSUE TITLE
Indentation is not checked with respect of the indentation of the previous line

### ISSUE THREAD

**Issue by luisesanmartin** [worldbank/stata-linter#52]
A line like this:
```{stata}
    *No indentation after three slashes
    iebaltab weight price, ///
    grpvar(tmt) pairoutput("p") ftestoutput("p") replace 
```
is not flagged as a bad practice, even when the last line should have extra indentation. This is because there is indentation already at the beginning of that line, though it is at the same level as the line containing the `///` characters. In case the previous line was also indented, extra-indentation should be checked in the line next to `///`
